source/Engine.cpp

```diff
@@ -266,12 +266,13 @@ void Engine::Place()
 		if(!ship->IsParked() && ship->GetSystem())
 			ships.push_back(ship);
 	
-	// Add NPCs to the list of ships. Fighters have to be assigned to carriers,
-	// and all but "uninterested" ships should follow the player.
 	shared_ptr<Ship> flagship = player.FlagshipPtr();
 	
 	// Update the active NPCs for missions based on the player's conditions.
 	player.UpdateMissionNPCs();
+	
+	// Add NPCs to the list of ships. Fighters have to be assigned to carriers,
+	// and all but "uninterested" ships should follow the player.
 	for(const Mission &mission : player.Missions())
```

Comment:
Why move the comment?

Reply:
Because the comment should be directly above the code block that it describes. I think it most accurately fits here, since only calls to `Place` actually add NPCs to the ship list.

Reply:
This comment is about putting npcs into flight and assigning them the correct parent pointer, which the above code lines contribute to.